salt/repositories/containers.repo

```diff
@@ -4,4 +4,9 @@ type=rpm-md
 gpgcheck=0
 enabled=1
 autorefresh=1
-baseurl=http://download.opensuse.org/repositories/Virtualization:/containers/openSUSE_Leap_{{ grains['osrelease'] }}/
\ No newline at end of file
+{% if grains['oscodename'].startswith("openSUSE Leap") %}
+baseurl=http://download.opensuse.org/repositories/Virtualization:/containers/openSUSE_Leap_{{ grains['osrelease'] }}/
+{% else %}
+# not really important: it will not be used
```

Comment:
I don't know even if it's worth taking this into account. Having deployed this new code with `SLE` happened on the transition period only. I would ignore completely the `SLE` case since we aren't supporting it. I'd leave this whole block as `baseurl=http://download.opensuse.org/repositories/Virtualization:/containers/openSUSE_Leap_{{ grains['osrelease'] }}/`